aait/web/group-4/starter-project/components/common/Footer.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+const Footer: React.FC = () => {
+  let socialMedia: string[] = [
+    "/images/common/facebook.png",
+    "/images/common/linkedin.png",
+    "/images/common/instagram.png",
+    "/images/common/twitter.png",
+    "/images/common/youtube.png",
+  ];
+  return (
+    <footer className="flex flex-col pt-16 border-t-2 px-20 pb-10 font-{montserrat} ">
```

Comment:
Use kabab-case naming convention for CSS classes. change `font-{montserrat}` to `font-montserrat`

Reply:
for some reason it doesn't work when I change it to font-montserrat